src/DataValues/QuantityValue.php

```diff
@@ -473,14 +342,22 @@ public function getArrayValue() {
 	 * Constructs a new instance of the DataValue from the provided data.
 	 * This can round-trip with @see getArrayValue
 	 *
+	 * @note: if the upperBound or lowerBound field is missing from $data,
+	 * this returns an UnboundedQuantityValue, not a QuantityValue!
+	 *
 	 * @since 0.1
 	 *
 	 * @param mixed $data
 	 *
-	 * @return self
+	 * @return UnboundedQuantityValue
```

Comment:
Callers should only care that the result is guaranteed to be an UnboundedQuantityValue, and not guaranteed to be a QuantityValue.